app/views/home/show.html+redesign.erb

```diff
@@ -0,0 +1,180 @@
+<%= open_graph_tags title: t(".title_og"), turbo_native_title: t(".turbo_native_title"), description: t(".description_og") %>
+
+<div class="bg-navy">
+  <div class="flex items-start justify-center max-w-6xl mx-auto px-8 py-48 -mt-24 bg-gradient-radial from-navy-gradient via-navy to-navy sm:-mt-48 sm:pt-72">
+    <div class="hidden flex-shrink-0 h-80 w-80 bg-gray-300 lg:block xl:h-96 xl:w-96">
+      <%# Image placeholder %>
+    </div>
+    <div class="mt-4 lg:ml-16 xl:ml-28">
+      <h1 class="text-3xl tracking-tight font-heading text-white lg:text-4xl xl:text-5xl">
+        <%= t(".hero_title_1") %>
+        <span class="text-teal"><%= t(".hero_title_accent") %></span>
+        <%= t(".hero_title_2") %>
+      </h1>
+      <p class="mt-3 text-sm text-white font-light leading-8 md:text-base md:mt-5">
+        <%= t(".subtitle_1") %>
+        <span><%= t(".subtitle_2") %></span>
+        <span class="md:block"><%= t(".subtitle_3") %></span>
+      </p>
+      <div class="mt-5 md:mt-8">
+        <%= link_to new_role_path, class: "relative inline-flex items-center px-10 py-4 font-heading text-typography text-lg bg-green rounded-full font-semibold hover:bg-green-hover focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-navy focus:ring-white" do %>
+          <%= t(".get_started") %>
+          <%= inline_svg_tag "icons/solid/arrow_long_right.svg", class: "text-typography ml-3 -mr-1 h-6 w-6", aria_hidden: true %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="max-w-6xl mx-auto my-24 sm:my-32">
+  <div class="pb-5 px-4 w-full">
+    <h2 class="text-3xl tracking-tight text-typography font-heading text-center lg:text-4xl xl:text-5xl">
+      <%= t(".reviews_header") %>
+    </h2>
+    <p class="mt-3 text-sm text-typography text-light text-center leading-8 md:text-base md:mt-5 lg:leading-10">
+      <%= tag.span(t(".reviews_subtitle_1"), class: "block") %>
+      <%= tag.span(t(".reviews_subtitle_2"), class: "block") %>
+    </p>
+  </div>
+  <div class="flex flex-col space-y-12 items-center px-2 mt-20 md:px-8 lg:items-stretch lg:flex-row lg:space-y-0 lg:space-x-12">
+    <div class="flex flex-col w-full">
+      <div class="flex-grow border-2 p-12 border-green">
+        <div class="flex">
+          <% 5.times do %>
+            <span>
+              <%= inline_svg_tag "icons/solid/star.svg", class: "text-teal h-6 w-6", aria_hidden: true %>
+            </span>
+          <% end %>
+          <span class="sr-only">5 star rating</span>
+        </div>
+        <p class="mt-4 mx-1 text-sm text-typography leading-relaxed font-light">
+          Switching jobs and overcoming the period between contracts was extremely painful. <span class="font-bold">railsdevs.com is precisely what I needed</span>. Now companies come to me instead of me chasing the work. I am incredibly grateful Joe did such a favor to the Rails community and built this platform.
```

Comment:
Let's extract the name, title, and copy to a YAML file so we can easily make changes in the future. We are already using `config/testimonials.yml` for business testimonials. Let's move that to `config/testimonials/businesses.yml` and create a new `developers.yml` file for these.